### ISSUE TITLE
Do not run pr pipeline in merge queue

### ISSUE THREAD

**Issue by afeefghannam89** [NETWAYS/ansible-collection-elasticstack#193]
The pr workflow runs by every pull request, which has a suitable name `fix/*` and `feature/*`. We do not need to run this workflow again while putting a pull request in the merge queue. No new code come between the two actions.

**Comment by widhalmt** [comment]
Right. That's a shortcoming of merge queues. I gave feedback to GitHub that I want to have the possibility to run some mandatory checks *only* during the merge queue. But it seems this will still need some time or maybe it isn't realized at all.
